README.md

```diff
@@ -48,14 +48,6 @@ pip3 install -e '.[jax_gpu]' -f 'https://storage.googleapis.com/jax-releases/jax
 pip3 install -e '.[full]'
 ```
 
-*TL;DR to install the PyTorch version for GPU run:*
```

Comment:
@andres-fr  We don't want to remove this command from the README. Could you add it back please?

Reply:
I'm so sorry!  You are right, I shouldn't have removed that part. I added it back